ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java

```diff
@@ -316,6 +316,7 @@ public static boolean prepareImport(boolean isImportCmd,
       }
       inReplicationScope = true;
       tblDesc.setReplWriteId(writeId);
+      tblDesc.setOwnerName(tblObj.getOwner());
```

Comment:
have to see if this works for import/export user case where the ownership required might be different that the replication use case.

Reply:
its done only for repl flow